### ISSUE TITLE
definitions cause chefspec failures

### ISSUE THREAD

**Issue by rmcleod8** [zuazo/ssh_authorized_keys-cookbook#14]
### Cookbook Version
0.5.0

### Chef Client Version
chef_version=14.6.47

### Platform Details
platform=redhat
platform_version=7.3

### Scenario
ChefSpec unit test that executes a recipe which includes the `ssh_authorize_key` resource

### Steps to Reproduce
* Install chefdk 3.4.38
* create a recipe that writes a file or anything
* add a block to that recipe which uses the `ssh_authorize_key` resource. Make sure the `validate_key` option is set to false
* create a unit test which checks that the file was written or whatever you specified
* rspec spec/unit/recipes/your_recipe_spec.rb

### Expected Result
The unit test passes and correctly evaluates the `ssh_authorize_key` resource by loading the definition files

### Actual Result
When unit testing a recipe using chefspec, if the tested recipe includes the `ssh_authorize_key` resource, the test will fail with

```
NoMethodError:
       undefined method `params' for nil:NilClass
```

This is likely related to Chefspec, however, this cookbook is using a definitions file which I believe is deprecated chef functionality per https://github.com/chefspec/chefspec/issues/926#issuecomment-418837313

**Comment by mrmarbury** [comment]
is there a fix or a workaround for this?

**Comment by zuazo** [comment]
This should be fixed by #15. Relesed in [`1.0.0`](https://github.com/zuazo/ssh_authorized_keys-cookbook/releases/tag/1.0.0).
